viewer/packages/api/src/public/RevealManager.ts

```diff
@@ -91,6 +86,10 @@ export class RevealManager {
     this._pointCloudManager.resetRedraw();
   }
 
+  public setCameraInMotion(inMotion: boolean): void {
+    this._cameraInMotion = inMotion;
+  }
```

Comment:
You can concider listening to the events directly here instead as this would technically allow things that have access to RevealManager to set this regardless if the camera actually is in motion or not, but not going to block on this. Up to you what you think is best.